gnomad_constraint/utils/generic.py

```diff
@@ -244,3 +244,34 @@ def remove_unnecessary_variants(
         exome_crit &= exome_ht.freq[freq_index].AF <= af_cutoff
     exome_ht = exome_ht.filter(exome_crit)
     return context_ht, exome_ht
+
+
+def get_all_pop_lengths(
+    ht, pops: Tuple[str], prefix: str = "observed_", skip_assertion: bool = False
+):
+    """
+    Get the minimum array length for specific per population annotations in `ht`.
+
+    The annotations are specified by the combination of `prefix` and each population in `pops`.
+
+    :param ht: Input Table.
```

Comment:
might be helpful to specfiy what this Table is (for example, you specify the input HT is a coverage HT in the model functions)